met_connectors/src/frost/spatial.rs

```diff
@@ -97,22 +98,30 @@ fn parse_polygon(polygon: &Polygon) -> String {
 fn json_to_spatial_cache(
     resp: serde_json::Value,
     timestamp: DateTime<Utc>,
-) -> Result<SpatialCache, Error> {
+) -> Result<DataCache, Error> {
     let data = extract_data(resp, timestamp)?;
 
     let lats: Vec<f32> = data.iter().map(|d| d.1.latitude).collect();
     let lons: Vec<f32> = data.iter().map(|d| d.1.longitude).collect();
     let elevs: Vec<f32> = data.iter().map(|d| d.1.elevation).collect();
-    let values: Vec<f32> = data.iter().map(|d| d.0.body.value).collect();
+    let values: Vec<Option<f32>> = data.iter().map(|d| Some(d.0.body.value)).collect();
 
-    Ok(SpatialCache::new(lats, lons, elevs, values))
+    Ok(DataCache::new(
+        lats,
+        lons,
+        elevs,
+        Timestamp(0),
+        RelativeDuration::minutes(5),
+        0,
+        vec![values; 1],
```

Comment:
This shape is wrong, right? I think you want the transpose, like we did in harness.rs. Something like `values.iter().map(|v| vec[v; 1]).collect()`